package.json

```diff
@@ -57,7 +57,7 @@
         "@atlaskit/tree": "^4.1.11",
         "@atlaskit/width-detector": "^0.3.1",
         "@babel/cli": "^7.2.3",
-        "@babel/core": "^7.3.4",
+        "@babel/core": "^7.0.0-0",
```

Comment:
is this a downgrade? Why is this necessary?

Reply:
the other babel plugins like `plugin-proposal-class-properties, plugin-proposal-object-rest-spread, preset-typescript` etc have dependency to this babel/core version and it was throwing error while installing our package.